apps/src/templates/CodeWorkspaceContainer.jsx

```diff
@@ -98,8 +98,7 @@ const styles = {
     borderLeftStyle: 'none',
     borderTopWidth: 1,
     borderTopStyle: 'solid',
-    borderTopColor: '#ddd',
-    overflow: 'hidden'
```

Comment:
My only concern here is unexpected side effects with Blockly. I assume you have already verified that droplet is ok with this change. If you've tested various Blockly levels as well, I'm ok.

Reply:
Thanks for noting that. I went ahead and spot-checked a couple blockly levels, and they look good.